MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,1085 @@
+</tr>
+
+<tr>
+<td>{{'[ColumnSpan](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItemBase.html#Syncfusion_XForms_DataForm_DataFormItemBase_ColumnSpan)'| markdownify }}</td>
+<td>ColumnSpan</td>
+<td>Gets or sets the number of column that this item spans to increase item's width.</td>
+</tr>
+
+<tr>
+<td>{{'[IsVisible](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItemBase.html#Syncfusion_XForms_DataForm_DataFormItemBase_IsVisible)'| markdownify }}</td>
+<td>IsVisible</td>
+<td>Gets or sets a value indicating whether the data form item layout (Holds editor and label) is visible or not.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>RowOrder</td>
+<td>Gets or sets the row order of the editor in the data form.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>Padding</td>
+<td>Gets or sets the space between the data form item’s editor and label.</td>
+</tr>
+</table>
+
+### DataFormItem
+
+<table>
+
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th>
+<th>Description</th></tr>
+
+<tr>
+<td>{{'[GroupName](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_GroupName)'| markdownify }}</td>
+<td>GroupName</td>
+<td>Gets or sets the name of the group to the editor.</td>
+</tr>
+
+<tr>
+<td>{{'[Name](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_Name)'| markdownify }}</td>
+<td>FieldName</td>
+<td>Gets or sets the editor's name in the data form.</td>
+</tr>
+
+<tr>
+<td>{{'[LabelText](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_LabelText)'| markdownify }}</td>
+<td>LabelText</td>
+<td>Gets or sets the editor’s label text.</td>
+</tr>
+
+<tr>
+<td>{{'[PlaceHolderText](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_PlaceHolderText)'| markdownify }}</td>
+<td>PlaceholderText</td>
+<td>Gets or sets the prompt string displayed or the water mark to provide hint for users in the editors.</td>
+</tr>
+
+<tr>
+<td>Nil</td>
+<td>PlaceholderColor</td>
+<td>Gets or sets the color of prompt string displayed or the water mark to provide hint for users in the editors.</td>
+</tr>
+
+<tr>
+<td>{{'[ShowLabel](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_ShowLabel)'| markdownify }}</td>
+<td>ShowLabel</td>
+<td>Gets or sets a value indicating whether to show the label to the editor.</td>
+</tr>
+
+<tr>
+<td>{{'[IsReadOnly](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.DataFormItem.html#Syncfusion_XForms_DataForm_DataFormItem_IsReadOnly)'| markdownify }}</td>
+<td>IsReadOnly</td>
+<td>Gets or sets a value indicating whether editor can be enabled or disabled.</td>
```

Comment:
<td>Gets or sets a value indicating whether the editor can be enabled or disabled.</td>